tools/garaga_rs/wasm-pack-build-and-patch

```diff
@@ -0,0 +1,193 @@
+    const input = fs.readFileSync(tsFile, 'utf8')
+      .replace(/\bexport\s+default\s+[^;]+;/, '');
+    const output =
+      exportFunc + '\n\n' +
+      input;
+    fs.writeFileSync(tsFile, output, 'utf8');
+  }
+
+  // creates .mjs file:
+  // - uses .wasm.mjs file as the default WASM module
+  // - adds and exports the init() function
+  // - replaces the default WASM load behavior via URL by direct use
+  // - removes the default export
+  {
+    const importStmt = 'import default_module_or_path from \'./' + wasmmjsName + '\';';
+    const exportFunc = 'export function init() { return __wbg_init({ module_or_path: default_module_or_path }); }';
+    const input = fs.readFileSync(jsFile, 'utf8')
+      .replace('new URL(\'' + wasmName + '\', import.meta.url)', 'default_module_or_path')
+      .replace(/\bexport\s+default\s+[^;]+;/, '');
+    const output =
+      importStmt + '\n\n' +
+      exportFunc + '\n\n' +
+      input;
+    fs.writeFileSync(mjsFile, output, 'utf8');
+  }
+
+  // creates .cjs file:
+  // - uses .wasm.cjs file as the default WASM module
+  // - adds and exports the init function
+  // - replaces the default WASM load behavior via URL by the direct use
+  // - removes the default export
+  // - transpiles ESM style exports to CommonJS style exports
+  {
+    const importStmt = 'const default_module_or_path = require(\'./' + wasmcjsName + '\');';
+    const exportFunc = 'exports.init = init;\nfunction init() { return __wbg_init({ module_or_path: default_module_or_path }); }';
+    const input = fs.readFileSync(jsFile, 'utf8')
+      .replace('new URL(\'' + wasmName + '\', import.meta.url)', 'default_module_or_path')
+      .replace(/\bexport\s+default\s+[^;]+;/, '');
+    const output =
+      '\'use strict\';' + '\n\n' +
+      importStmt + '\n\n' +
+      exportFunc + '\n\n' +
+      transpileExports(input);
+    fs.writeFileSync(cjsFile, output, 'utf8');
+  }
+
+  // encodes .wasm file as .wasm.cjs:
+  // - creates .wasm.cjs exporting the binary contents of the .wasm file
+  {
+    const input = fs.readFileSync(wasmFile).toString('base64');
+    const output =
+      '\'use strict\';' + '\n\n' +
+      'module.exports = "data:application/wasm;base64,' + input + '";';
+    fs.writeFileSync(wasmcjsFile, output, 'utf8');
+  }
+
+  // encodes .wasm file as .wasm.mjs:
+  // - creates .wasm.mjs exporting the binary contents of the .wasm file
+  {
+    const input = fs.readFileSync(wasmFile).toString('base64');
+    const output =
+      'export default "data:application/wasm;base64,' + input + '";';
+    fs.writeFileSync(wasmmjsFile, output, 'utf8');
+  }
+
+  // cleans up now obsolete files:
+  // - removes .js, .wasm, and .wasm.d.ts files
+  {
+    fs.rmSync(jsFile);
+    fs.rmSync(wasmFile);
+    fs.rmSync(wasmtsFile);
+  }
+}
+
+function build() {
+  // executes wasm-pack in the current folder
+  const command = 'wasm-pack build --target web --release';
```

Comment:
it seems the --no-default-features flag is missing here to avoid pyo3 dep when building (leading to a pyo3 error)